composer.json

```diff
@@ -31,7 +31,6 @@
 		"nette/utils": "^3.0",
 		"latte/latte": "^3.0",
 		"tracy/tracy": "^2.4",
-		"dg/adminer-custom": "^1.8"
```

Comment:
Do not need to remove adminer, if someone needs it, just move it to `require-dev`.